scanpy/plotting/_tools/scatterplots.py

```diff
@@ -1066,15 +1078,16 @@ def _basis2name(basis):
 def _process_image(adata, data_points, img_key, crop_coord, scale_spot, bw=False):
     offset = 100
     cmap_img = None
-    img = adata.uns['images'][img_key]
+    library_id = list(adata.uns['spatial'].keys())[0]
```

Comment:
Why is this always the first one?

Same with all the other places

Reply:
Because is the only one, the dict is structured like this:
```python
{"spatial":{"library_id":{#everything else}}}
```

Reply:
Btw, this is also related to the proposed version of how concatenation should work in theislab/anndata#295

Reply:
But isn't the point of concatenation that there might two?

Reply:
Right yes. 
I definitely will have to rethink how this is handled for plotting. For now I've decided to just take the first one if no library_id is given. Had to add an additional argument in embedding for that. Should work but I'm also not very happy on the default behaviour, but for now it's fine I think.